Gems/Atom/Feature/Common/Code/Source/CoreLights/DirectionalLightFeatureProcessor.cpp

```diff
@@ -1341,6 +1345,11 @@ namespace AZ
             {
                 const float invShadowmapSize = 1.0f / GetShadowmapSizeFromCameraView(handle, segmentIt.first);
 
+                Vector3 previousAabbMin;
+                Vector3 previousAabbMax;
```

Comment:
super nit: maybe initialize these with CreateZero? I can see that they are not actually used until cascadeIndex == 1, so they're getting set before they're used. But it took me a minute to realize that, and I think for a person reading this code for the first time, they'd be more trusting if this was initialized they way previousNear and previousFar are initialized.

Reply:
👍 No real issues doing the init since this is like once per frame max.